src/args.jl

```diff
@@ -2255,3 +2264,27 @@ function _splitdef!(blk, value_args, key_args)
     end
     blk
 end
+
+
+# const _attrsymbolcache = Dict{Symbol, Dict{Symbol, Symbol}}()
```

Comment:
I think you can remove that commented block ?

Reply:
done!